kotlin/vci-client/src/main/java/io/mosip/vciclient/authorizationCodeFlow/interactiveAuthorization/handler/InteractiveAuthorizationHandler.kt

```
@@ -0,0 +1,174 @@
+                        endpoint,
+                        traceabilityId
+                    )
+
+                else ->
+                    throw InteractiveAuthorizationException("Unsupported interaction type: $type")
+            }
+
+        } catch (e: InteractiveAuthorizationException) {
+            logger.warning("Interactive authorization failed: ${e.message}")
+            throw e
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)